### ISSUE TITLE
Interested in co-maintainers?

### ISSUE THREAD

**Issue by zgreen** [charliekassel/vuejs-datepicker#679]
Hi @charliekassel 👋 Thanks so much for creating and maintaining this. Just trying out/looking through this plugin for the first, and it seems there are some outstanding pull requests that would be great to get merged. Do you need/want assistance, co-maintainers, etc.? This is a popular project and it would be great if it could stay up-to-date.

For now, I've created https://github.com/zgreen/vuejs-datepicker/tree/build that merges #611 and #626 (folks that need it can install it like this `npm i zgreen/vuejs-datepicker#build`). Obviously this isn't ideal; if you're looking for help on this project, let us know!

And thanks again, it's super helpful to get Vue projects up and running when components like this already exist.

**Comment by charliekassel** [comment]
Hi @zgreen - it's true this project has been neglected as I've not had time for it recently, I've not even worked with Vue for over a year now.
Thank you for the offer, I will give it some thought.

**Comment by Gregg** [comment]
Whoa, yes, this is awful that this project is being neglected.  Please merge #611 and #626 ASAP or approve others to work in this project.  My students are going through my tutorials now and things are not working.  =(

**Comment by charliekassel** [comment]
@Gregg - I'd appreciate you not talking to me like that.
I provide this code as an open source project, you are welcome to use it but have no right to demand what should or should not be done with it.
If you are teaching people how to code I'd suggest you either write a library with them or choose another library to use.

Be respectful to those that have put work into projects so that others can benefit from it.

And please don't send me personal email.

**Comment by Gregg** [comment]
Whoa...  My comment wasn't meant as an attack, but it sounds like you might have taken it as one.

I feel some responsibility for the Vue community.  I want to ensure that the ecosystem is healthy and popular libraries work.  This is a great library and I do appreciate all you've put into it.  I probably should have started with that feeling in my comment, sorry about that.  I am grateful for your work.   

You said yourself that "the project has been neglected."

If you don't have time to maintain this project, but care about it continuing on to help the Vue community, it seems like you might consider allowing someone else to take over?

**Comment by Gregg** [comment]
I regret using such strong language in my frustration about this project.  It wasn't very friendly.  Sorry about that.

**Comment by wmcmurray** [comment]
**I am interested** in being added to contributors for this project :raising_hand_man: !

Because we use this module a lot, and we have found some bugs, so I've submitted some PRs but they never got merged, so now we have a custom branch in our dependencies as well to incorporate those fixes in production :grimacing: 

I suppose it's normal to be hesitant to let stranger put their hands in our code and to trust them..
(I would be !) but I think it's better for the project than holding it back :wink:

**Comment by fessacchiotto** [comment]
Hello! Any suggestions of another vue.js component which supports a "typeable" feature and  it is widely supported by the community? Thank you!

**Comment by simllll** [comment]
Hi all,
as I had several issues with this library, and there was no update the last 6 months, I looked into it over the weekend and did the following things:
* Upgrade all libraries
* uses vue-eslint-parser for eslint.
* Fixed SSR support for NuxtJS
* Merged #611: show calendar on foucs prop
* Merged #626: fixes missing close and open events
* Based on #536: Allow custom types, I changed the method to getTypedDate which should return a fully parsed Date object. 
* Added a beforeDateInput slot.

I have created a PR with all the fixes included, and also publsihed a NPM package @hokify/vuejs-datepicker.
Please test it and let me know, @charliekassel would b great if you find some time to look over it and consider merging it into this repo.

regards
Simon

**Comment by caryevans** [comment]
This is a really great project @charliekassel - thanks for all your work on it. I've spent a lot of hours wrestling with it. I think it is great, but it has some bugs. I noticed that there are a lot of pull requests, it looks like it is not being updated very frequently. I've been in your situation before where I ultimately let the project die on the vine and later regretted it. 

You have people that are willing to help. I would encourage you to accept their help and make this picker the de-facto standard picker for Vue projects. You've done all the heavy lifting, but there are a ton of edge-cases to chase down which are no fun to fix but really important. This is where all the help comes in. 

You've started the snowball... let's make this thing awesome together.
